api/qsharp/Microsoft.Quantum.Core.yml

```diff
@@ -25,9 +25,9 @@ functions:
   summary: Returns the integer that specifies how the next value of a range is calculated.
 newtypes:
 - uid: microsoft.quantum.core.attribute
-  summary: Compiler recognized attribute used to mark user defined types as attributes.
+  summary: Compiler-recognized attribute used to mark user-defined types as attributes.
```

Comment:
It'd be a lot easier for me if one of the maintainers just took care of merging/transferring these changes over to those repos. (I was gonna say "blech, but sure," until I noticed that it wasn't even _one_ separate repo needing changes, it was _two_ separate repos! That's a high enough barrier I'm happy to make it someone else's problem.)

Reply:
I think is fair @quuxplusone, I'll try to implement them. Thank you again. I'll keep this PR open until I make those changes in their respective repos.

@cgranade, would it be possible to create a bidirectional flow between the API documentation and the `*.yml` files? It's not the first time we have this problem with external contributions and I don't know if a warning or a tip in the contribution guide will be enough to prevent this issue from happening again.

Reply:
> I think it is fair @Quuxplusone, I'll try to implement them. Thank you again. I'll keep this PR open until I make those changes in their respective repos.

Thanks for taking point on this, @geduardo!

> @cgranade, would it be possible to create a bidirectional flow between the API documentation and the `*.yml` files? It's not the first time we have this problem with external contributions and I don't know if a warning or a tip in the contribution guide will be enough to prevent this issue from happening again.

Unfortunately, I don't think it will be possible to go from `*.yml` files back to the original `*.qs` sources. It might be good to set up a GitHub Action to automatically comment on PRs that modify `*.yml` files. I also remember @natke mentioning the possibility of separating out the AI reference into its own repo, that may be good to follow up on as well. (+@KittyYeungQ for docs impact)